File: Hashtable_Angular/src/app/presentation/controller/AppController.ts
import {Table} from "../../dto/Table";
import {Service} from "../../service/Service";

export class AppController{
  private _model: Table = new Table(10);
  private _service : Service = new Service(this._model);
  constructor() {
  }


  get model(): Table {
    return this._model;
  }

  set model(value: Table) {
    this._model = value;
  }

  get service(): Service {
    return this._service;
  }

  set service(value: Service) {
    this._service = value;
  }
}

File: Hashtable_Flutter/lib/presentation/controller/AppController.dart

import 'package:demo/model/Table.dart' as t;
import 'package:demo/service/Service.dart';

class AppController {

  t.Table _model = t.Table(10) ;
  Service _service = Service(t.Table(10));

  AppController();

  Service getService() => _service;

  void setService(Service service) => _service = service;

  t.Table getModel() => _model;

  t.Table setModel(t.Table table) => _model = table;
}
